less/pages/homepage.less

```diff
@@ -0,0 +1,130 @@
+/*!
+ * Copyright (c) 2021 Eclipse Foundation, Inc.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * Contributors:
+ *   Martin Lowe <martin.lowe@eclipse-foundation.org>
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+*/
+
+section.active-events {
+  & {
+    margin-top: 40px;
+    margin-bottom: 0;
+  }
+  table {
+    font-size: 16px;
+  }
+  table tbody tr td {
+    padding: 16px 8px 16px 0;
+  }
+  @media (min-width: 992px) {
+    & {
+      margin-top: 0;
+    }
+  }
+}
+
+section.about-home > .container {
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
+
+// TODO replace with cards-component where possible
+section.past-events {
+  & {
+    background-image: url('/images/first-banner.png');
+    background-size: cover;
+    margin-bottom: 30px;
+    padding: 40px 0px;
+  }
+  .event-container {
+    margin-top: 20px;
+  }
+  .event-panel {
+    & {
+      border-radius: 1em;
+      border: none;
+      overflow: hidden;
+      padding: 0;
+      height: 100%;
+      text-decoration: none;
+      box-shadow: 4px 6px 9px 0px rgba(0, 0, 0, 0.31);
+    }
+    &:hover {
+      text-decoration: underline;
+    }
+    .panel-heading {
+      padding: 0;
+    }
+    .panel-heading img {
+      min-width: 100%;
+    }
+    .panel-body {
+      padding: 1em;
+      padding-top: 0;
+    }
+  }
+}
+section.host-jakartaone {
+  & {
+    background-image: url('/images/host-banner.jpg');
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    background-color: #282e70;
+    border-top: 7px #282e70 solid;
+    border-bottom: 7px #282e70 solid;
+  }
+
+  > .container {
+    margin-top: 50px;
+    margin-bottom: 50px;
+  }
+  @media (min-width: 992px) {
```

Comment:
Maybe we can make the media nested in "> .container" above? As it only works for the container.

Reply:
as a rule, I don't tend to nest things that would mean there are less than 3 selector rulesets together like that. It leads to a lot of not really needed nesting and makes it somewhat harder to read rather than clarifying it